### PR TITLE
Feature/15/button component

### PR DESCRIPTION
**Pull request by henryadamsgit**
Completed buttons

### CODE REVIEW

File: src/pages/WelcomePage/WelcomePage.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @use "/src/styles/_variables.scss" as *;
 @use "/src/styles/_typography.scss" as *;
+@use "../../components/Button/Button.scss" as *;
```
Comment (olirob93) [inline src/pages/WelcomePage/WelcomePage.scss]: Does this need to be here?

File: src/pages/WelcomePage/WelcomePage.jsx [hunk 2]
```diff
@@ -3,12 +3,13 @@ import "./WelcomePage.scss";
...
 import Logo from "../../assets/images/logo black.svg";
 import LoginForm from "../../components/Forms/LoginForm/LoginForm";
+import Button from "../../components/Button/Button";
 import { signInWithEmailAndPassword } from "firebase/auth";
 import { auth } from "../../firebase";
 import { useState } from "react";
 import { useNavigate } from "react-router-dom";
 
-const WelcomePage = ({ setUser }) => {
+const WelcomePage = ({ setUser, buttonStyle }) => {
```
Comment (olirob93) [inline src/pages/WelcomePage/WelcomePage.jsx]: Button style doesn't need to be in the props does it?

File: src/pages/WelcomePage/WelcomePage.jsx [hunk 3]
```diff
@@ -48,7 +49,13 @@ const WelcomePage = ({ setUser }) => {
             emailInput={setEmail}
             passwordInput={setPassword}
           />
-          <button onClick={getUser}>Log in</button>
+          <Button
+            className={buttonStyle}
```
Comment (olirob93) [inline src/pages/WelcomePage/WelcomePage.jsx]: classname can be removed

File: src/components/HomepageCard/HomepageCard.jsx
Comment (olirob93) [inline src/components/HomepageCard/HomepageCard.jsx:1]: button style can be removed from all places in this component

File: src/components/HomepageCard/HomepageCard.scss
Comment (olirob93) [inline src/components/HomepageCard/HomepageCard.scss:1]: do you need the scss imported here

File: src/pages/Booking/Booking.jsx
Comment (olirob93) [inline src/pages/Booking/Booking.jsx:1]: button style can be removed from everywhere in this file

File: src/pages/Booking/Booking.scss
Comment (olirob93) [inline src/pages/Booking/Booking.scss:1]: remove this scss we shouldn't have put the buttons in unless it was replacing it from a component

File: src/pages/Clients/Clients.jsx
Comment (olirob93) [inline src/pages/Clients/Clients.jsx:1]: remove the button from here

File: src/pages/Resources/Resources.jsx
Comment (olirob93) [inline src/pages/Resources/Resources.jsx:1]: remove the button from here

File: src/pages/Settings/Settings.jsx
Comment (olirob93) [inline src/pages/Settings/Settings.jsx:1]: remove the button from here

File: src/pages/Staff/Staff.jsx
Comment (olirob93) [inline src/pages/Staff/Staff.jsx:1]: remove the button from here

**Review by olirob93 — CHANGES_REQUESTED** [review]
Looks good, I think I might have confused you when I said have you put the button where it needs to.

It should only be used where we already have components using a button. We don't want to add the button in random places as it will be used when we create other components.

File: src/pages/Booking/Booking.scss
Comment (henryadamsgit) [reply src/pages/Booking/Booking.scss:1]: Remove all scss or just the button import?

File: src/components/HomepageCard/HomepageCard.scss [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (olirob93) [inline src/components/HomepageCard/HomepageCard.scss]: whtie space

File: src/pages/Booking/Booking.jsx
Comment (olirob93) [inline src/pages/Booking/Booking.jsx:1]: nothing should be different in this file

File: src/pages/Booking/Booking.scss [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (olirob93) [inline src/pages/Booking/Booking.scss]: still has a line

File: src/pages/Resources/Resources.jsx [hunk 6]
```diff
@@ -4,11 +4,11 @@ import SideNav from "../../components/SideNav/SideNav";
...
 const Resources = () => {
   return (
-  <div>
-    <h1>Resources</h1>
-    <SideNav/> 
-  </div>
-  )
+    <div>
+      <h1>Resources</h1>
+      <SideNav />
```
Comment (olirob93) [inline src/pages/Resources/Resources.jsx:9]: no space here for sideNav

**Review by olirob93 — CHANGES_REQUESTED** [review]
formatting issues

**Review by olirob93 — APPROVED** [review]
all good

**Pull request merged**